packages/react/react/src/components/Separated/Separated.tsx

```diff
@@ -2,9 +2,9 @@
 /** @jsxImportSource react */
 import { Children, Fragment, isValidElement, PropsWithChildren, ReactNode } from 'react';
 
-type Props = {
+interface Props extends PropsWithChildren {
   with: ReactNode;
-} & PropsWithChildren;
+}
```

Comment:
Yes, it operates identically in terms of functionality. However, other files in the same directory use 'interface' instead of the 'type' keyword for declaring Props, so I made this modification for consistency. Furthermore, the TypeScript official documentation recommends using 'interface' and suggests switching to 'type' only in case issues arise
Thx
> https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
I didn't know. That's nice!!